content/blog/aws-cloudwatch-made-easy-with-pulumi-infrastructure-as-code/index.md

```diff
@@ -137,7 +137,7 @@ of the health of this app:
         ],
     });
 
-The [Crosswalk for AWS API for dashboards](https://github.com/pulumi/pulumi-awsx/tree/master/nodejs/awsx/cloudwatch#dashboards)
+The [Crosswalk for AWS API for dashboards]({{< relref "/docs/guides/crosswalk/aws/cloudwatch/#defining-cloudwatch-dashboards-in-code" >}})
```

Comment:
I am not sure if we want to change the link here. The link text seems to specifically talk about the API for dashboards. If we do want to link to the guide instead, that's fine. Just making sure that that is what we should be doing.

Reply:
I could also change the link text?  @justinvp  do you have an opinion here?

Reply:
This link should not change. The original link is to the `awsx` README, which provides higher-level capabilities on top of `aws`, and this link is specifically about "Crosswalk for AWS" which is what `awsx` is.

Reply:
Done, thank you both!